pkg/ebpf/tracee.go

```diff
@@ -271,6 +302,14 @@ func New(cfg Config) (*Tracee, error) {
 	}
 	t.containers = c
 
+	if t.config.RequiredInits.kallsyms {
+		t.kernelSymbols, err = helpers.NewKernelSymbolsMap()
+		if err != nil {
+			t.Close()
```

Comment:
Instead of failing tracee in case of an error initilializing the symbols map, let's just print an error for all the events that have such dependencies, and continue as usual with any other event

Reply:
As we discussed, this will be answered in another PR